rtrans-esb/src/main/java/ca/bc/gov/moh/esb/util/audit/AuditProcessor.java

```diff
@@ -56,18 +56,17 @@ public AuditProcessor() {
         persistenceUnitProperties.put("javax.persistence.jdbc.url", appProperties.getProperty(DATABASE_URL));
         persistenceUnitProperties.put("javax.persistence.jdbc.user", appProperties.getProperty(DATABASE_SCHEMA));
         persistenceUnitProperties.put("javax.persistence.jdbc.password", appProperties.getProperty(DATABASE_PASSWORD));
-        
-        emf = Persistence.createEntityManagerFactory(PERSISTENCE_UNIT_RTRANS_ESB_AUDITS, persistenceUnitProperties);
+        //emf = Persistence.createEntityManagerFactory(PERSISTENCE_UNIT_RTRANS_ESB_AUDITS, persistenceUnitProperties);
     }
 
     public <T> T insert(T record) {
-        EntityManager em = emf.createEntityManager();
-        EntityTransaction et = em.getTransaction();
-
-        et.begin();
-        em.persist(record);
-        et.commit();
+        // EntityManager em = emf.createEntityManager();
+        // EntityTransaction et = em.getTransaction();
 
+        // et.begin();
+        // em.persist(record);
+        // et.commit(); 
+        logger.info("Skipping entering database records ");
```

Comment:
We probably don't need a logger at info level every time an audit event happens. 
The better approach to skipping audit might just be to remove or comment the 
`.process(AUDIT_PROCESSOR)` lines from the actual routes

Reply:
updated code to not call auditing.